lib/utils.js

```diff
@@ -56,6 +56,13 @@ module.exports.fileCommand = function (notifier, options, cb) {
   });
 };
 
+module.exports.fileCommandForWin = function (notifier, options, cb) {
```

Comment:
Why is this used instead of just `fileCommand`? E.g. it breaks using the CLI (and probably regular invocation). But using `fileCommand` instead works with the cli on both cmd.exe, powershell, cmder, git bash.